src/styles/card.css

```
@@ -0,0 +1,86 @@
+.card {
+  font-family: "Roboto", Arial, sans-serif;
+  width: 200px;
+  padding-top: 32px;
+  padding-left: var(--horizontal-spacing);
+  padding-right: var(--horizontal-spacing);
+  padding-bottom: var(--vertical-spacing);
```

This can be simplified to oneliner:
```suggestion
  padding: 32px var(--horizontal-spacing) var(--horizontal-spacing);
```